modules/howtos/examples/kv_operations.py

```diff
@@ -1,240 +1,145 @@
 from datetime import timedelta
-document = {"foo":"bar", "bar":"foo"}
-result = collection.insert("document-key", document, expiry=timedelta(seconds=5))
-#end::insert_w_options[]
-"""----
-
-Setting a Compare and Swap (CAS) value is a form of optimistic locking - dealt with in depth in the xref:concurrent-document-mutations.adoc[CAS page].
-Here we just note that the CAS is a value representing the current state of an item; each time the item is modified, its CAS changes.
-    The CAS value is returned as part of a document’s metadata whenever a document is accessed.
-    Without explicitly setting it, a newly-created document would have a CAS value of _0_.
-
-                                                                                          _Timeout_ is an optional parameter which in the .NET SDK has a type value of `TimeSpan`.
-    Timeout sets the timeout value for the underlying network connection.
-We will add to these options for the _Replace_ example:
-
-    [source,python]
-    ----
-"""
-#tag::replace[]
-document = {"foo":"bar","bar":"foo"}
-result = collection.replace("document-key", document, cas = 12345, expiry = timedelta(minutes=1))
-#end::replace[]
-"""
-----
-
-Expiration sets an explicit time to live (TTL) for a document, for which you can also xref:sdk-xattr-example.adoc[see a more detailed example of TTL discovery] later in the docs.
-    We'll discuss modifying `Expiration` in more details xref:#net-modifying-expiration[below].
-For a discussion of item (Document) _vs_ Bucket expiration, see the
-xref:6.5@server:learn:buckets-memory-and-storage/expiration.adoc#expiration-bucket-versus-item[Expiration Overview page].
-
-[source,python]
-                       ----
-"""
-# tag::expiration[]
-document = {"foo": "bar", "bar": "foo"}
-result = collection.upsert("document-key", document, cas=12345, expiry=timedelta(minutes=1),
-                           durability=ClientDurability(ReplicateTo.ONE, PersistTo.ONE))
-# end::expiration[]
-"""
-----
-
-Here, we have add _Durability_ options, namely `PersistTo` and `ReplicateTo`.
-    In Couchbase Server releases before 6.5, Durability was set with these two options -- see the xref:https://docs.couchbase.com/python-sdk/2.5.5/durability.html[6.0 Durability documentation] -- covering  how many replicas the operation must be propagated to and how many persisted copies of the modified record must exist.
-    Couchbase Data Platform 6.5 refines these two options, with xref:synchronous-replication.adoc[Synchronous Replication] -- although they remain essentially the same in use -- as well as adding the option of xref:transactions.adoc[atomic document transactions].
 
-The following example demonstrates using the newer durability features available in Couchbase server 6.5 onwards.
-
-[source,python]
-----
-"""
-# tag::upsert_syncrep[]
-from couchbase.durability import Durability
+from couchbase.durability import Durability, ServerDurability, \
```

Comment:
I'd prefer parentheses for grouping the multiline import. Though it doesn't seem that pep8 mandates anything, https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses suggests it's preferred?

(But I'm a bit out of touch with python dev at the mo, so ignore this if it's perfectly idiomatic!)

Reply:
Mmmmm, thats fair. Parentheses would be better (although I see it both ways seemingly just as often).  I'll make an update :)